com.unity.render-pipelines.high-definition/Documentation~/Dynamic-Resolution.md

```diff
@@ -20,84 +25,91 @@ The policy type can be one of the following:
 - `DynamicResScalePolicyType.ReturnsPercentage`:  The DynamicResolutionHandler expects the `scaler` to return a screen percentage value. The value set will be clamped between the minimum and maximum percentage set in the HDRP Asset.
 - `DynamicResScalePolicyType.ReturnsMinMaxLerpFactor`:  The DynamicResolutionHandler expects the `scaler` to return a factor `t` that is the [0 ... 1] range and that will be used as a lerp factor between the minimum and maximum screen percentages set in the HDRP asset.
 
-The example below shows how to call this function. In a real production environment, you would call this function depending on the performance of your application.
-
+It is best practice to keep the screen percentage above 50%. However, HDRP supports values up to 5, 100.
```

Comment:
Here's, I believe it's oddly phrased. 
I would say something like, "HDRP supports values from 5% up to 100%"